src/Apps/MyCollection/Routes/EditArtwork/Utils/useCreateOrUpdateOrDeleteArtwork.ts

```diff
@@ -1,15 +1,18 @@
 import { MyCollectionCreateArtworkInput } from "__generated__/useCreateArtworkMutation.graphql"
 import { MyCollectionUpdateArtworkInput } from "__generated__/useUpdateArtworkMutation.graphql"
+import { MyCollectionDeleteArtworkInput } from "__generated__/useDeleteArtworkMutation.graphql"
 import { useCreateArtwork } from "../Mutations/useCreateArtwork"
 import { useUpdateArtwork } from "../Mutations/useUpdateArtwork"
+import { useDeleteArtwork } from "../Mutations/useDeleteArtwork"
 
 export type ArtworkInput =
   | MyCollectionCreateArtworkInput
   | MyCollectionUpdateArtworkInput
 
-export const useCreateOrUpdateArtwork = () => {
+export const useCreateOrUpdateOrDeleteArtwork = () => {
```

Comment:
I wonder if it makes sense to do this change here. We could use `const { submitMutation: deleteArtwork } = useDeleteArtwork()` directly in [MyCollectionArtworkForm.tsx](https://github.com/artsy/force/pull/10834/files#diff-7445b7d2b9d4df4ded453cf7511b68c1234ad7d6168cabd72b81d75309fa2cce)`. I've just added this helper to upudate-or-create an artwork with one function call.

Reply:
I don't like the idea of having requests in separate files. I think it makes sense to have everything in one place
What if I keep upudate-or-create function and add another one only for deletion to the same file?

Reply:
It seems to be the preferred pattern in Force to use the `useMutation` pattern and than use `commitMutation` directly in the component that uses the file ([see comment from @damassi ](https://github.com/artsy/force/pull/10732#discussion_r942810460)).

Reply:
Aaaah, alright, I didn't know
Changing to the preferred way

Reply:
Thanks 🙇